frontend/app/player/MessageDistributor/managers/DOM/FocusManager.ts

```diff
@@ -0,0 +1,26 @@
+import logger from 'App/logger';
+import type { SetNodeFocus } from '../../messages';
+import type { VElement } from './VirtualDOM';
+import ListWalker from '../ListWalker';
+
+
+const FOCUS_CLASS = "-openreplay-focus"
+
+export default class FocusManager extends ListWalker<SetNodeFocus> {
+	constructor(private readonly vElements:  Map<number, VElement>) {super()}
```

Comment:
you can remove {super()} for empty constructors iirc

Reply:
net, typescript is rugaetsya